web-common/src/components/floating-element/WithTogglableFloatingElement.svelte

```diff
@@ -4,6 +4,7 @@
 
   import { FloatingElement } from "./index";
   import Portal from "../Portal.svelte";
+  export let disabled = false;
```

Comment:
Can you give insight into why this was needed? It seems to me if a float element needs to be disabled, you can control that at the trigger point (usually the primary slot that uses `toggleFloatingElement`)

Reply:
Ah, good catch. We were using this before for the selectors but seems like it's not needed now.